multimodal/src/autogluon/multimodal/presets.py

```diff
@@ -1,7 +1,5 @@
 from typing import List, Optional
 
-from ray import tune
```

Comment:
Did you test uninstalling ray and running AutoMM? This should work so long as you aren't using HPO.

Reply:
Tested. It works without doing HPO.